tests/Schema/BlueprintTest.php

```diff
@@ -77,6 +77,23 @@ public function testDropTable(): void
         );
     }
 
+    public function test_dropIfExists(): void
+    {
+        $conn = $this->getDefaultConnection();
+
+        $blueprint = new Blueprint('Test3', function (Blueprint $table) {
+            $table->dropIfExists();
+        });
+
+        $queries = $blueprint->toSql($conn, new Grammar());
+        $this->assertEquals(
+            'drop table if exists `Test3`',
+            $queries[0]
+        );
+
+        self::assertTrue($conn->statement($queries[0]));
```

Comment:
In the test method, it would be better to unify either `$this` or `self`.

```suggestion
        $this->assertTrue($conn->statement($queries[0]));
```